setup.cfg

```diff
@@ -107,12 +107,30 @@ max-line-length = 99
 ignore =
     N802  # function name should be lowercase
     W503  # line break occurred before a binary operator
+    D100  # public module needs docstring
+    D101  # public class needs docstring
+    D102  # public method needs docstring
+    D103  # public function needs docstring
+    D104  # public package needs docstring
+    D105  # magic method needs docstring
+    D106  # nested class needs docstring
+    D107  # __ini__ needs docstring
 exclude =
     .tox
     build
     dist
     .eggs
     docs/conf.py
+rst-roles=
```

Comment:
Since this project is 99% python code, why don't we also allow default roles (without py:)? I personally find tedious to always write the prefix.

Reply:
agreed